tests/integration/synapseclient/test_tables.py

```diff
@@ -227,6 +228,21 @@ def test_materialized_view(syn, project):
     assert all(view_df.columns == ["F.Name", "F.Born", "F.Hipness", "F.Living", "P.Name", "P.Age"])
 
 
+def test_dataset(syn, project):
+    cols = [Column(name='id', columnType='ENTITYID'),
+            Column(name='name', columnType='STRING')]
+
+    dataset = Dataset(
+        name="Pokedex",
+        parent=project,
+        datasetItems=[{'entityId': "syn20685093", 'versionNumber': 1}],
```

Comment:
what is this entity id?

Reply:
It's to a sample file (which I realize is not viewable by the general public). Is there a list of official entities I can use for testing? Happy to just share the sample file as well.